lix/packages/sdk/src/merge/merge.test.ts

```diff
@@ -6,47 +6,40 @@ import type {
 	NewChange,
 	NewConflict,
 	NewSnapshot,
+	Snapshot,
 } from "../database/schema.js";
 import type { LixPlugin } from "../plugin.js";
+import { createPhantomSnapshot } from "../query-utilities/create-phantom-snapshot.js";
 
 test("it should copy changes from the sourceLix into the targetLix that do not exist in targetLix yet", async () => {
 	const mockSnapshots = [
-		{
-			id: "sn1",
-			value: { id: "mock-id", color: "red" },
-		},
-		{
-			id: "sn2",
-			value: { id: "mock-id", color: "blue" },
-		},
-		{
-			id: "sn3",
-			value: { id: "mock-id", color: "green" },
-		},
+		createPhantomSnapshot({ id: "mock-id", color: "red" }),
```

Comment:
Why phantom snapshot? 

- unclear what phantom is referring to 
- unclear that it's a JSON snapshot (and not a blob)


```suggestion
		createJsonSnapshot({ id: "mock-id", color: "red" }),
```